hw/top_earlgrey/dv/cov/conn_otp_ctrl_ast_obs_ext_volt.el

```diff
@@ -0,0 +1,28 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//==================================================
+// This file contains the Excluded objects
+// Generated By User: root
+// Format Version: 2
+// Date: Fri Nov 11 23:06:36 2022
+// ExclMode: default
+//==================================================
+CHECKSUM: "1202717726 1014860158"
+INSTANCE: tb.dut.top_earlgrey
+ANNOTATION: "[LOW_RISK] Covered via connectivity. Cannot be covered in DV due to behavioral models of AST and OTP."
+Toggle otp_ext_voltage_h_io "net otp_ext_voltage_h_io"
+CHECKSUM: "708483626 3106053973"
+INSTANCE: tb.dut.top_earlgrey.u_otp_ctrl
+ANNOTATION: "[LOW_RISK] Covered via connectivity. Cannot be covered in DV due to behavioral models of AST and OTP."
```

Comment:
should we say.. "must be covered in vendor closed source DV"?

Reply:
Sure, done.